convex/util/ollama.ts

```diff
@@ -0,0 +1,154 @@
+import { Ollama } from 'langchain/llms/ollama';
+import {
+  ChatCompletionContent,
+  CreateChatCompletionRequest,
+  LLMMessage,
+  retryWithBackoff,
+} from './openai';
+import { IterableReadableStream } from 'langchain/dist/util/stream';
+
+const ollamaModel = process.env.OLLAMA_MODEL || 'llama2';
+export const UseOllama = process.env.OLLAMA_HOST !== undefined;
```

Comment:
I think you might need to update the `Setting up local inference` instructions in the README to mention ollama host being required.